includes/polling/cisco-qfp.inc.php

```diff
@@ -101,7 +101,7 @@
         if (! empty($util_data) && ! empty($memory_data)) {
             $total_packets = $util_data[$util_oids['InTotalPps']] + $util_data[$util_oids['OutTotalPps']];
             $throughput = $util_data[$util_oids['InTotalBps']] + $util_data[$util_oids['OutTotalBps']];
-            $average_packet = $throughput / 8 / $total_packets;
+            $average_packet = floor($throughput / 8 / ($total_packets ?: 1));
```

Comment:
shouldn't it be something like this?
```suggestion
            $average_packet = $total_packets ? ($throughput / 8 / $total_packets) : 0;
```

Reply:
Well, the only annoying thing about it is that the stupid component storage spams the eventlog.

Reply:
Added floor